### ISSUE TITLE
Confusion over which version is the latest on NuGet.

### ISSUE THREAD

**Issue by BentTranberg** [elmish/Elmish.WPF#82]
I wanted to revert to using Cmd.ofMsg without the warning, after reading something about this somewhere, so I opened NuGet in VS 2019 to check what was available. There was nothing, which was a bit mysterious, because I did remember reading it was supposed to be available.

NuGet says I've got "Installed: 2.0.0-beta-9" and in the unopened dropdown below "Version: Latest prerelease 2.0.0-beta-9". It also says "Date published: Saturday, March 23, 2019 (3/23/2019)"

When I open the dropdown with versions, there's a list, going like this:

Latest prerelease 2.0.0-beta-9
- - - - - - - - - - - -
2.0.0-beta-9
2.0.0-beta-8
...
2.0.0-beta-3
2.0.0-beta-2
2.0.0-beta-11
2.0.0-beta-10
2.0.0-beta-1
1.0.0-beta-7
1.0.0-beta-6
...

The 10 and 11 betas are in fact the latest versions, judging from the dates.

In other words, in this case NuGet doesn't understand which version is supposed to be the latest, so people are not notified of the latest updates. I had to look for it in the dropdown, and it was a bit of luck that I did. I think this should be fixed.

I would guess one possible solution is to change the naming scheme of the betas. How about going to 2.0.1-beta-01, to leave space for 99 correctly sorted betas in 2.0.1 ?

**Comment by cmeeren** [comment]
Yes, I noticed that, and it's a problem. But now that Elmish is out of beta, I will likely be releasing 2.0.0 final very soon. Will try to do it later today.
